memo_app.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'json'
+require 'securerandom'
+
+MEMOS_PATH = File.join(settings.public_folder, 'memos.json')
+
+helpers do
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+end
+
+get '/' do
+  unless File.exist?(MEMOS_PATH)
+    File.open(MEMOS_PATH, 'w') do |file|
+      file.write([].to_json)
+    end
+  end
+
+  memos = JSON.parse(File.read(MEMOS_PATH))
+  erb :index, locals: { memos: }
+end
+
+get '/memos/new' do
+  erb :new
+end
+
+post '/memos' do
+  id = SecureRandom.uuid
+  title = params[:title]
+  content = params[:content]
+
+  memo = { id:, title:, content: }
```

Comment:
一度ローカル変数にいれる意味があまりないかなと思いました。

```suggestion
  memo =
    {
      id: SecureRandom.uuid,
      title: params[:title],
      content: params[:content],
    }
```

Reply:
アドバイス、ありがとうございます。
おっしゃるとおり、ローカル変数にする意味（例えば、複数回使うなど）がありませんので、ご提案のように修正したいと思います。